application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -147,10 +157,17 @@ private void rawTag(@NotNull SlashCommandEvent event) {
         if (tagSystem.handleIsUnknownTag(id, event)) {
             return;
         }
-
-        event.replyEmbeds(MessageUtils.generateEmbed(null,
-                MessageUtils.escapeMarkdown(tagSystem.getTag(id).orElseThrow()), event.getUser(),
-                TagSystem.AMBIENT_COLOR, null))
+        /*
+         * MessageUtils.generateEmbed(null,
+         * MessageUtils.escapeMarkdown(tagSystem.getTag(id).orElseThrow()), event.getUser(),
+         * TagSystem.AMBIENT_COLOR, null)
+         */
+        event
+            .replyEmbeds(new EmbedBuilder()
+                .setDescription(MessageUtils.escapeMarkdown(tagSystem.getTag(id).orElseThrow()))
+                .setAuthor(String.valueOf(event.getUser()))
+                .setColor(TagSystem.AMBIENT_COLOR)
+                .build())
```

Comment:
You forgot the timestamp `setTimestamp(Instant.now())`

Reply:
Timestamp is there now but the `setAuthor` thing has to be a `setFooter` in the way it was beforehand.